lib/ansible/modules/monitoring/zabbix_group.py

```diff
@@ -66,6 +66,12 @@
         required: false
         default: None
         version_added: "2.1"
+    validate_certs:
+        description:
+            - If False, SSL certificates will not be validated. This should only be used on personally controlled sites using self-signed certificates.
+        required: false
+        default: True
+        version_added: "2.4"
```

Comment:
wouldn't it be better to add to a common doc_fragment? seems like other options would also benefit from this to avoid repetition